### ISSUE TITLE
Suggestion: Color tile support in Windows 10 desktop version

### ISSUE THREAD

**Issue by vpontin** [saenzramiro/rambox#1325]
According to [this website](https://needs-be.blogspot.com.br/2016/08/windows-10-visual-elements-incl-start.html), making a file named [exefilename].visualelementsmanifest.xml is possible to add colors to a pinned desktop app (aka exe) in start menu/screen. Like this:

![image](https://user-images.githubusercontent.com/3485505/33232301-e14dc4ca-d1ea-11e7-9272-1d67fb288596.png)
In the image, "Windows 8" can be interpreted as a pinned exe without the xml (it uses the current windows theme colors), and "Windows 8.1" shows a customized color for the tile using the xml.

If possible to add a companion visualelementsmanifest.xml to make the pinned rambox icon more appealing, it would be very nice.

More documentation about this (the MS ones are more focused on UWP apps, but some parameters are valid for desktop apps too):

https://msdn.microsoft.com/en-us/library/windows/apps/xaml/dn449733.aspx
https://needs-be.blogspot.com.br/2016/08/windows-10-visual-elements-incl-start.html
https://www.askvg.com/tip-customize-start-screen-tiles-background-color-text-color-and-logo-in-windows-8-1/
https://superuser.com/questions/962849/how-can-i-change-the-icon-size-on-the-windows-10-start-menu (see the "Original Post" section in the approved answer)

**Comment by saenzramiro** [comment]
Good idea, I will take a look at this for a future release. Hopefully, Electron supports this.
